README.md

```diff
@@ -1,2 +1,122 @@
+`sysup [-check | -update]`
+- Offline Usage:
+`sysup [-check | -update] -updatefile IMG_FILE [-updatekey KEY_FILE]`
+- Managing Trains:
+`sysup [-list-trains | -change-train TRAIN]`
+
+## Primary Arguments (Required)
+Only **one** of these arguments may be used at a time.
+- **-check**
+   - Check for updates
+- **-update**
+   - Start performing updates
+- **-list-trains**
+   - List all the pkg "trains" that the system is aware of (defined in "/usr/local/etc/sysup.json")
+- **-change-train TRAIN_NAME**
+   - Reconfigure the package repository files to point to the designated TRAIN_NAME.
+   - ***WARNING*** This will remove *all* package repository configuration files on the system and create a single "/etc/pkg/train.conf" file containing the configuration for the desired package train.
+
+## Daemonizing the updater
+- **-websocket**
+   - Startup a websocket service for direct API access and events
+   - This is a primary argument that should not be combined with any other flags except possibly `-addr`
+- **-addr ADDRESS**
+   - Websocket service address (IP:portnumber). This is a general option for all primary arguments to allow it to talk to a currently-running websocket service
+   - Default value: "127.0.0.1:8134"
+   
+## Secondary/Optional Arguments
+There are a number of secondary/optional flags that can be used for additional functionality:
+
+### Offline Updates
+sysup allows for the possibility of offline updates via an image file containing the packages from a repository. This will pull all the required packages for the update from the image file instead of trying to download packages from an online repository.
+- **-updatefile IMG_FILE**
+   - Run in offline mode with the designated "IMG_FILE" used as the package repository.
+   - This can be used with both "-check" and "-update" primary arguments.
+   - **NOTE** This ignores any trains or package repositories that are configured on the system.
+- **-updatekey KEY_FILE**
+   - When doing offline updates, use the "KEY_FILE" as the public SSL key to verify of the integrity of the IMG and packages.
+   - Default value: none. Will not verify SSL signature of the packages.
+   
+### Additional Update Options
+These arguments are add-ons for the "-update" argument and are typically not needed for standard use
+- **-bename NAME**
+   - Use "NAME" for the new boot environment that will be created.
+   - A boot environment with "NAME" must *not* already exist, otherwise sysup will return an error.
+   - Default Value: sysup with automatically generate a unique boot environment name using a date/time stamp.
+      - Example of auto-generated BE name: "2018-11-27-14-34-26"
+- **-fullupdate**
+   - Force a "full" update of all packages (including kernel/world).
+   - Default Value: This is automatically determined based on whether the base packages (kernel/world) are tagged as newer on the package repository.
+- **-stage2**
+   - Start stage2 of an update (installing non-base packages)
```

Comment:
Should be (Installing non-kernel package updates)

Reply:
Fixed!